examples/error_estimates_forces.jl

```diff
@@ -53,7 +53,7 @@ tol = 1e-8;
 
 # We compute the reference solution ``P_*`` from which we will compute the
 # references forces.
-scfres_ref = self_consistent_field(basis_ref; tol, callback=info->nothing)
+scfres_ref = self_consistent_field(basis_ref; tol, callback=identity)
```

Comment:
not the same

Reply:
Nothing in particular, but we might want to extend the API in the future (eg so that the callback can request an early stopping)

Reply:
Actually we moved from `info->nothing` to `identity` some time ago to allow chaining of callbacks. Therefore the change is actually reasonable.